pkg/operator/clustermembercontroller/clustermembercontroller.go

```diff
@@ -86,18 +86,19 @@ func (c *ClusterMemberController) sync(ctx context.Context, syncCtx factory.Sync
 func (c *ClusterMemberController) reconcileMembers(recorder events.Recorder) error {
 	unhealthyMembers, err := c.etcdClient.UnhealthyMembers()
 	if err != nil {
-		return err
+		return fmt.Errorf("could not get list of unhealthy members: %v", err)
 	}
 	if len(unhealthyMembers) > 0 {
 		klog.V(4).Infof("unhealthy members: %v", spew.Sdump(unhealthyMembers))
+		// TODO: Why not return error here?
```

Comment:
@hexfusion could we return an error here, or is there a specific reason why we don't?

Reply:
This should return an error I think we were maybe worried about going degraded on possible known conditions during scale-up but I think the information would be useful. +1